{
 "bad": [
  "Exception",
  "ImportError",
  "RuntimeError",
  "ValueError",
  "ZeroDivisionError",
  "__name__",
  "caller",
  "dict",
  "f",
  "float",
  "functools.partial",
  "isinstance",
  "len",
  "list",
  "logging.NullHandler",
  "min",
  "mock.MagicMock",
  "range",
  "sum",
  "tuple"
 ],
 "deps": [
  [
   "retry-0.9.2-py_0"
  ]
 ]
}